feature_engine/feature_builder.py

```diff
@@ -91,14 +96,15 @@ def featurize(self, col: str="message") -> None:
         # Step 1. Preprocess the relevant column (the column that has the text used to create the features).
         self.preprocess_chat_data(col=col)
         # Step 2. Set Conversation Data Object.
-        self.set_self_conv_data()
+        # TODO This will throw an error with downstream tasks
```

Comment:
What is this TODO with the error?

Reply:
This is where we add conversation num to the conv features. Adding them here threw an error in my current pipeline of lexical feature processing. I have added in the conversation num to the conv features in a subsequent function that avoids the error.

I left the TODO here so that I remember to add in preprocessing for conv nums as discussed in the Thursday's meeting (Have separate scripts/notebooks cleaning individual datasets and calling them from the pipeline).